src/Mirza/Common/Beam.hs

```diff
@@ -44,7 +44,7 @@ textType = BMigrate.DataType pgTextType
 
 -- | Field definition to use for last updated columns
 lastUpdateField :: BMigrate.TableFieldSchema PgColumnSchemaSyntax (Maybe LocalTime)
-lastUpdateField = field "last_update" (maybeType timestamptz) (defaultTo_ (B.just_ now_))
+lastUpdateField = field "last_update" (maybeType timestamp) (defaultTo_ (B.just_ now_))
```

Comment:
You need to convert `now()` to UTC

`now() AT TIME ZONE 'UTC'`

Reply:
The triggers are added in the function `Mirza.Common.Utils.addLastUpdateTriggers`, and I have made the `NOW() AT TIME ZONE 'UTC'` change there